Makefile

```diff
@@ -1,14 +1,26 @@
+ifndef DEADFISH_DOCKER_NOSUDO
+dockercmd=sudo docker
+else
+dockercmd=docker
+endif
```

Comment:
This doesn't really change much, besides adding a bit of universality. It's ok, I understand the concept (and hey, it's a contribution, thanks :D), but I'm wondering whether things that anyway require additional local changes to work should be in the repo. Maybe a script for making docker work without sudo would make it more full. Idk, calling in @capturcus, what do you think?

Reply:
The main benefit of this patch is that my local changes are restricted just to `~/.bashrc`. I only need to define an environment variable once, and suddenly I don't need to use sudo anymore. Otherwise I have to modify the `Makefile` and watch out in order not to include changes to it when doing `git commit`.

As for fixing the need to use sudo for docker... I don't think we can create a script that will work on all environments. The fix probably depends on the configuration of your OS.

Reply:
Running docker with or without sudo depends on your system configuration and I want to enable both choices. This change does exactly that.